packages/web-components/fast-ssr/webpack.fast-style.config.cjs

```diff
@@ -0,0 +1,36 @@
+const path = require("path");
+const srcDir = path.resolve(__dirname, "./src");
+const outDir = path.resolve(__dirname, "./dist");
+
+/**
+ * This file exports the minified fast-style web component
+ */
+module.exports = {
+    entry: {
```

Comment:
I'm curious if we really need webpack for this, or if we could just use `tsc` to compile to JS and the `html` file imports that JS file?

Reply:
So this depends, I did this purposefully because with `tsc` we're treating it like a regular part of the library, and it imports `DOM` since we're using ES6 so it's not capable of being standalone. I suppose this really is down to how we're going to integrate it into the renderer.

Reply:
ahh yeah okay. Cool it's fine then, I was just hoping to reduce deps and build processes (and maintenance over time)

Reply:
For sure, I did try that first, but alas, imports.

Reply:
Updated, removed the `DOM` dependency and changed the mimetype for the server and changed script type to module..